test/info.py

```diff
@@ -26,42 +26,46 @@
+    (   0x240,      'k20dx_frdmk64f_if',                        'k20dx_bl',         'FRDM-K64F'                     ),
+    (   0x250,      'k20dx_frdmkl02z_if',                       'k20dx_bl',         None                            ),
+    (   0x260,      'k20dx_frdmkl26z_if',                       'k20dx_bl',         None                            ),
+    (   0x280,      'k20dx_frdmkw24f_if',                       'k20dx_bl',         None                            ),
+    (   0x824,      'lpc11u35_lpc824xpresso_if',                None,               'LPCXpresso824-MAX'             ),
+    (   0x1017,     'lpc11u35_hrm1017_if',                      None,               'mbed-HRM1017'                  ),
+    (   0x1018,     'lpc11u35_ssci824_if',                      None,               'Switch-Science-mbed-LPC824'    ),
+    (   0x1019,     'lpc11u35_sscity_if',                       None,               'Switch-Science-mbed-TY51822r3' ),
+    (   0x1050,     'lpc11u35_lpc812xpresso_if',                None,               'NXP-LPC800-MAX'                ),
+    (   0x1060,     'lpc11u35_lpc4088qsb_if',                   None,               'EA-LPC4088'                    ),
+    (   0x1062,     'lpc11u35_lpc4088dm_if',                    None,               'EA-LPC4088-Display-Module'     ),
+    (   0x1070,     'sam3u2c_nrf51mkit_if',                     'sam3u2c_bl',       'Nordic-nRF51822'               ),
+    (   0x1090,     'k20dx_rbl_if',                             'k20dx_bl',         'RedBearLab-nRF51822'           ),
+    (   0x1095,     'k20dx_rblnano_if',                         'k20dx_bl',         'RedBearLab-BLE-Nano'           ),
+    (   0x1100,     'sam3u2c_nrf51dk_if',                       'sam3u2c_bl',       'Nordic-nRF51-DK'               ),
+    (   0x1114,     'lpc11u35_ssci1114_if',                     None,               'LPC1114FN28'                   ),
+    (   0x1120,     'sam3u2c_nrf51dongle_if',                   'sam3u2c_bl',       'Nordic-nRF51-Dongle'           ),
+    (   0x1234,     'lpc11u35_c027_if',                         None,               'u-blox-C027'                   ),
+    (   0x2015,     'lpc11u35_efm32gg_stk_if',                  None,               None                            ),
+    (   0x5050,     'lpc11u35_arm_watch_stm32f411_if',          None,               None                            ),
+    (   0x5051,     'lpc11u35_arm_watch_efm32_if',              None,               None                            ),
+    (   0x5052,     'lpc11u35_arm_watch_nrf51_if',              None,               None                            ),
+    (   0x7402,     'lpc11u35_6LoWPAN_BorderRouterHAT_if',      None,               'mbed-6LoWPAN-Border-Router-HAT'),
+    (   0x7403,     'lpc11u35_6LoWPAN_BorderRouterUSB_if',      None,               'mbed-6LoWPAN-Border-Router-USB'),
+    (   0x7404,     'lpc11u35_6LoWPAN_BorderRouterPOE_if',      None,               'mbed-6LoWPAN-Border-Router-POE'),
```

Comment:
The target name here needs to match the target name on the compile API. In this case the URL is https://developer.mbed.org/platforms/mbed-6LoWPAN-Border-Router-Ethernet/ so this should be changed to mbed-6LoWPAN-Border-Router-ETHERNET. With this change you should be able to use the compile API to build all the targets.